src/destructors.md

```diff
@@ -420,9 +420,9 @@ let x = (&temp()).use_temp();  // ERROR
 
 ## Not running destructors
 
-### `forget`
+### Manually preventing destructors
```

Comment:
This makes me wonder "preventing from what"?

I'd either way "Manually preventing destructors from running" or something like "Manually suppressing destructors" or so.